### ISSUE TITLE
Enable copy/paste with idos

### ISSUE THREAD

**Issue by tomkludy** [litchie/dospad#64]
I would like to copy/paste text from websites into idos but I am unable to.  There doesn't seem to be any paste-from-clipboard option.

**Comment by DominusExult** [comment]
no, neither Dospad/iDos nor the original Dosbox does allow copy/paste.
